.github/workflows/project-updater.yml

```
@@ -0,0 +1,23 @@
+name: Update projects when an issue/PR is created/labeled
```

Short and sweet? 
```suggestion
name: Update projects
```

---

Or at least

```suggestion
name: Update GH projects
```

to be clear which projects we're talking about